README.md

```diff
@@ -1,46 +1,21 @@
-# Getting Started with Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+# Debate Deck
+
+## Development to do list
+
+### Needed for MVP
+- Understand how redux works with state management
+- Figure out how to manage users
+- Integrate firebase authentication for login and signup
+- Decide data hierarchy/structure for database 
+- Use firebase database for storing card and project information.
```

Comment:
We should use a 100% free independent database like MySQL or MongoDB instead

Reply:
but hosting isn't free right? I don't have a PC at home I can host it from. Also, I tend to do my development on the web through code sandbox bc that's just what I'm used to. 

Also, I've tried making a node server but something inside of codesandbox blocks pulling from a node server, or I did something wrong. I could access the data in any web browser, but kept getting errors when I tried using the API within code sandbox. Plus, it's another thing to learn and I don't want to try and do everything bc then I might end up doing nothing. idk.

Reply:
on the flip side SQL data structure makes much more sense to me than all the NoSQL stuff.

Reply:
I agree with SQL data structure.

Reply:
I have several cloud servers and can deploy and set it up no problem. You wouldn't have to learn how to do it if you don't want to. Going this way might be slightly more work to setup (on my end), but the benefit would be that your project is platform agnostic. You could run and deploy it anywhere and not have to be stuck with Google services.

Reply:
"Also, I tend to do my development on the web through code sandbox bc that's just what I'm used to."

I highly recommend getting setup with a proper IDE like VS Code or Intellij IDEA for your coding. You will be amazed at the tooling, error handling, autocompletion, etc. and it will make your life so much easier in the long run.